scripts/release/release.js

```diff
@@ -0,0 +1,363 @@
+const { spawnSync } = require("child_process"),
+    readline = require("readline"),
+    fs = require("fs"),
+    util = require("util")
+;
+
+// Import third party library
+const gitRawCommits = require("git-raw-commits"),
+    semver = require("semver"),
+    getStream = require("get-stream"),
+    externalEditor = require("external-editor"),
+    _ = require("colors"),
+    { Octokit } = require("@octokit/rest"),
+    AWS = require('aws-sdk')
+;
+
+/*
+ * Required parameters
+ */
+// A valid github personal token that should have write access to github repo.
+const GITHUB_TOKEN=process.env.GITHUB_TOKEN;
+// A scaleway access key that should have write access to the devtool bucket.
+const SCW_ACCESS_KEY=process.env.SCW_ACCESS_KEY;
+// A scaleway secet key that should have write access to the devtool bucket.
+const SCW_SECRET_KEY=process.env.SCW_SECRET_KEY;
+
+
+/*
+ * Global configuration
+ */
+// The root git directory. All future path should be relative to this one.
+const ROOT_DIR = "../..";
+// The script used to build release binary
+const BUILD_SCRIPT = "./scripts/build.sh";
+// The directory that contains release binary created by BUILD_SCRIPT
+const BIN_DIR_PATH = "./bin/";
+// Path to README file
+const README_PATH = "./README.md";
+// Path to CHANGELOG file
+const CHANGELOG_PATH = "./CHANGELOG.md";
+// Go file that contain the version string to replace during release process.
+const GO_VERSION_PATH = "./cmd/scw/main.go";
+// Name of the temporary branch that will be used during the release process.
+const TMP_BRANCH = "new-release";
+// Name of the temporary remote that will be used during the release process.
+const TMP_REMOTE = "scaleway-release";
+// Name of the github repo namespace (user or orga).
+const GITHUB_OWNER = "jerome-quere";
+// Name of the github repo.
+const GITHUB_REPO = "scaleway-cli";
+// Name of the devtool bucket.
+const S3_DEVTOOL_BUCKET="scw-devtools";
+// Region of the devtool bucket .
+const S3_DEVTOOL_BUCKET_REGION="nl-ams";
+// S3 object name of the version file that should be updated during release.
+const S3_VERSION_OBJECT_NAME="scw-cli-v2-version";
+
+/*
+ * Usefull constant
+ */
+const GITHUB_CLONE_URL = `git@github.com:${GITHUB_OWNER}/${GITHUB_REPO}.git"`;
+const GITHUB_REPO_URL = `https://github.com/${GITHUB_OWNER}/${GITHUB_REPO}`;
+const COMMIT_REGEX = new RegExp(`${_typeReg.source}${_scopeReg.source}: *${_messageReg.source} *${_mrReg.source}`);
+const _typeReg = /(?<type>[a-zA-Z]+)/;
+const _scopeReg = /(\((?<scope>.*)\))?/;
+const _messageReg = /(?<message>[^(]*)/;
+const _mrReg = /(\(#(?<mr>[0-9]+)\))?/;
+
+
+async function main() {
+
+    /*
+     * Initialization
+     */
+
+    // Chdir to root dir
+    process.chdir(ROOT_DIR);
+
+    // Initialize github client
+    if (!GITHUB_TOKEN) {
+        throw new Error(`You must provide a valid GITHUB_TOKEN`)
+    }
+    octokit = new Octokit({ auth: GITHUB_TOKEN });
+
+    // Initialize s3 client
+    if (!SCW_ACCESS_KEY || !SCW_SECRET_KEY) {
```

Comment:
name of vars should be updated so we know it refers to `S3`

`(!AWS_S3_SCW_ACCESS_KEY || !AWS_S3_SCW_SECRET_KEY)` ?